pep-0712.rst

```diff
@@ -246,7 +246,27 @@ appear to be similar to this approach.
 This works well for fairly simple types, but leads to ambiguity in expected
 behavior for complex types such as generics. E.g. For ``tuple[int, ...]`` it is
 ambiguous if the converter is supposed to simply convert an iterable to a tuple,
-or if it is additionally supposed to convert each element type to ``int``.
+or if it is additionally supposed to convert each element type to ``int``. Or
+for ``int | None``, which isn't callable.
+
+Deducing the attribute type form the return type of the converter
```

Comment:
Minor typo: form -> from

Reply:
If you could monetize the amount of types I've done this typo, you and I could fund the PSF FOREVER